samples/javascript_typescript/12.nlp-with-luis/src/index.ts

```diff
@@ -48,7 +42,7 @@ const luisConfig = <ILuisService>botConfig.findServiceByNameOrId(LUIS_CONFIGURAT
 // Map the contents to the required format for `LuisRecognizer`.
 const luisApplication: LuisApplication = {
     applicationId: luisConfig.appId,
-    azureRegion: luisConfig.region,
+    endpoint: `https://${ luisConfig.region }.api.cognitive.microsoft.com`,
```

Comment:
This should instead be luisConfig.getEndpoint(). I'll fix this when I merge into master.